base_tier_validation/models/tier_validation.py

```diff
@@ -236,18 +248,6 @@ def _compute_is_reevaluation_required(self):
                 if len(valid_tiers) != len(rec.review_ids.definition_id):
                     rec.is_reevaluation_required = True
 
-    @api.model
-    def _calc_reviews_validated(self, reviews):
```

Comment:
Would it not be nice to keep this mechanism? It allows to take values for other fields into account when determining the validation status. See how https://github.com/OCA/server-ux/pull/1101 allows to define those fields as dependencies for the computed status in its `_get_validation_status_dependencies`.

Reply:
The fields used in https://github.com/OCA/server-ux/pull/1084 in the `_get_validation_status_dependencies()` method are already being used here as depends of the `_compute_validation_status()` method (and some more fields). IMO keeping this method (`_calc_reviews_validated()`) would be a mistake, it is no longer needed, the proper thing is to use the status (rather, the `validated` field), don't you think?